### ISSUE TITLE
Save and Load Optimizer/Scheduler state

### ISSUE THREAD

**Issue by guilherme439** [guilherme439/NuZero#7]
Implement methods for Save and Loading Optimizer State.
Allow training config option for either using the previous learning rate/scheduler or use the new one define in the config.

Difficulty -> medium
Priority -> high
